src/app/shared/autocomplete/autocomplete.component.html

```diff
@@ -0,0 +1,118 @@
+<div class="autocomplete-section">
+
+  <form [formGroup]="autocompleteForm">
+    <div class="form-group">
+      <label>City</label>
+      <input id="city"
+             type="text"
+             class="form-control"
+             formControlName="city"
+             (click)="toogleDropdown()"
+             autocomplete="off">
+      <div *ngIf="showDropdown" class="city-dropdown-container">
+        <div (click)="changeCity(city)" class="city-text" *ngFor="let city of citiesNames | searchFilter: getCityValue()">
+          {{city}}
+        </div>
+      </div>
+      <span *ngIf="!autocompleteForm.get('city').valid && autocompleteForm.get('city').touched"
+            class="text-muted">
+        Please enter a city!
+      </span>
+    </div>
+
+    <div class="autocomplete-row">
+      <div class="form-group autocomplete-col">
+        <label>Pick-up date</label>
+        <input id="pickUpDate"
+               type="Date"
+               class="form-control"
+               formControlName="pickUpDate">
+        <span *ngIf="!autocompleteForm.get('pickUpDate').valid && autocompleteForm.get('pickUpDate').touched"
+              class="text-muted">
+        Please enter a date!
+      </span>
+      </div>
+
+      <div class="form-group autocomplete-col">
+        <label>Pick-up time</label>
+        <input id="pickUpTime"
+               type="Time"
+               class="form-control"
+               formControlName="pickUpTime">
+        <span *ngIf="!autocompleteForm.get('pickUpTime').valid && autocompleteForm.get('pickUpTime').touched"
+              class="text-muted">
+        Please enter time!
+      </span>
+      </div>
+    </div>
+
+    <div class="autocomplete-row">
+      <div class="form-group autocomplete-col">
+        <label>Drop-off date</label>
```

Comment:
I think, autocomplete component should contain only autocomplete input.
Other fields we can put to widget.component.ts